dashboard/src/components/UpgradeForm/UpgradeForm.tsx

```diff
@@ -109,11 +117,6 @@ class UpgradeForm extends React.Component<IDeploymentFormProps, IDeploymentFormS
       <div>
         <form className="container padding-b-bigger" onSubmit={this.handleDeploy}>
           <div className="row">
-            <div className="col-8">
```

Comment:
this code was not reachable, `this.props.error` is checked before that is not set